spec/features/doctors/delete_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
```

Comment:
I mentioned this yesterday, feature tests should be modeled after the view page where that feature lives. There is no delete view page, so there should be no delete spec file. This feature is for the doctor's show page, so that is where these test should be!